packages/wdio-cucumber-framework/src/index.js

```diff
@@ -64,8 +64,10 @@ class CucumberAdapter {
              */
             this.getCurrentStep = ::this.cucumberReporter.eventListener.getCurrentStep
 
+            const featurePathsToRun = this.config.cucumberFeaturesWithLineNumbers ? this.config.cucumberFeaturesWithLineNumbers.toString().split(',') : this.specs
```

Comment:
In `ConfigParser.js` you have:
```js
        if (this._config.spec && isCucumberFeatureWithLineNumber(this._config.spec)) {
            this._config.cucumberFeaturesWithLineNumbers = this._config.spec.join(',')
        }
```
why do we first join the files and then split them here again?

Reply:
> Just out of curiosity: if you currently run e.g.
> 
> ```shell
> $ npx wdio run wdio.conf.js --spec ./featureA.feature --spec ./featureB.feature
> ```
> 
> with one capability set in the `wdio.conf.js` , it would spin up 2 sessions and would run both (1 feature file per session) in parallel. Looking at the current implementation it seems to me that now it would run 2 sessions but would run 2 feature files per session. Can you verify?

From what I can tell this is working as expected... 2 sessions spin up and there is 1 feature file worked on per session. Are you saying that you felt like 2 features would run per session? So 4 scenarios instead of 2?

Reply:
yeah, I haven't tried your branch locally yet but the current implementation feels like 2 sessions spin up running 2 feature files each (so 4 in total) instead of 1 feature file per session.

Reply:
I have not seen that locally. Shouldn't be an issue. Actually trying it right now. Will post results.

Reply:
The `split` issue has been resolved.